modules/apps/portal/portal-portlet-test/bnd.bnd

```diff
@@ -0,0 +1,3 @@
+Bundle-Name: Liferay Portal Portlet Test
+Bundle-SymbolicName: com.liferay.portal.portlet.test
```

Comment:
Hi @dantewang,
Why you are using "com.liferay.portal.portlet.test", rather than "com.liferay.portlet.test". Are there still conflicts? I saw you move the util class into your test module in later commit.

Tina